manifests/state-ofed-driver/0050_ofed-driver-ds.yaml

```diff
@@ -124,6 +128,10 @@ spec:
             - name: shared-doca-driver-toolkit
               mountPath: /mnt/shared-doca-driver-toolkit
             {{- end}}
+            {{- if .RuntimeSpec.UseDriversInventory }}
+            - name: drivers-inventory
```

Comment:
dont we want to always have these mounts ?

Reply:
we could also set it if  driver inventory env var is set to true  WDYT ?

Reply:
im ok with having this defined by default.

if we want this to be defined by the operator, perhaps path can be provided via template value ? or inferred from the relevant env var if it exists ?

lets see what other think.